samples/boards/esp32/flash_encryption/boards/yd_esp32.overlay

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2022 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&flash0 {
+	write-block-size = <32>;
```

Comment:
What is the reason for increasing the `write-block-size` for this particular board? Isn't it using the same module - and flash chip - as other devkits?

Reply:
I used the following file as a reference and I didn't pay attention to that detail about `write-block-size`:

https://github.com/zephyrproject-rtos/zephyr/blob/7fe5ab315a4453322780e5655364fe4de2a1b9b6/samples/boards/esp32/flash_encryption/boards/esp32.overlay#L1-L9

Recently this file has been renamed to `esp32_devkitc_wroom.overlay` [(Browse History)](https://github.com/zephyrproject-rtos/zephyr/commits/main/samples/boards/esp32/flash_encryption/boards/esp32_devkitc_wroom.overlay). The [ESP32-DEVKITC-WROOM](https://docs.zephyrproject.org/latest/boards/xtensa/esp32_devkitc_wroom/doc/index.html) board uses the same modules as the YD-ESP32 board. Not sure why the value was kept the same.

---

@sylvioalves ,

Can I remove the ` samples/boards/esp32/flash_encryption/boards/yd_esp32.overlay` file or do you know why the  `write-block-size` is being increased?

Reply:
@jcsxdev The `write-block-size` seems only to be used in the flash driver to align the write address. For now, I think you can leave it as is.

Reply:
Flash encryption feature requires block size of 32 bytes to work. So you can keep this.